keras_cv/utils/preset_utils.py

```diff
@@ -138,11 +153,22 @@ def check_preset_class(
 ):
     """Validate a preset is being loaded on the correct class."""
     config_path = get_file(preset, config_file)
-    with open(config_path) as config_file:
-        config = json.load(config_file)
+    try:
+        with open(config_path) as config_file:
+            config = json.load(config_file)
+    except:
+        raise ValueError(
+            f"The specified preset  `{preset}` is unknown. "
+            "Please check documentation to ensure the correct preset "
+            "handle is being used."
+        )
     cls = keras.saving.get_registered_object(config["registered_name"])
-    if not isinstance(classes, (tuple, list)):
+    # Subclass checking
+    if not issubclass(classes, (tuple, list)):
         classes = (classes,)
+    # Alias subclass checking
+    if any(issubclass(alias, cls) for alias in classes):
```

Comment:
I think we can just move this check down two lines. Always do subclass checking. And update KerasNLP to match.

Reply:
Yeah, combined the two into one check, thanks!